auth-api/migrations/alembic.ini

```diff
@@ -34,6 +34,11 @@ level = INFO
 handlers =
 qualname = alembic
 
+[logger_flask_migrate]
```

Comment:
Without this, it's hard to see any migration errors.

Reply:
+1